README.md

```diff
@@ -32,6 +32,7 @@ Name | Author | Description
 [srihash-cli](https://github.com/natemcmaster/srihash-cli) | [@natemcmaster](https://github.com/natemcmaster) | Generates the SRI hash for `<script>` tags in browsers [NuGet](https://nuget.org/packages/srihash-cli)
 [weeknumber](https://github.com/MarkusLund/weeknumber) | [@MarkusLund](MarkusLund) | Prints the current weeknumber to the command line. [NuGet](https://nuget.org/packages/weeknumber/)
 [dotnet-xdt](https://github.com/nil4/dotnet-transform-xdt) | [@nil4](https://github.com/nil4) | Global tool for applying XML Document Transformations to .NET configuration files, or any other XML-structured content. [NuGet](https://www.nuget.org/packages/dotnet-xdt/)
+[dmd5](https://github.com/Rwing/Dotnet.Tool.MD5) | [@Rwing](https://github.com/Rwing) | Just generate MD5 hash value in CLI. [NuGet](https://www.nuget.org/packages/dmd5)
```

Comment:
Can you put this higher in the list? I'm trying to keep this sorted alphabetically. Thanks!

Reply:
Okay, How does that look?